pages/candidate-intake.js

```diff
@@ -21,7 +20,7 @@ export default function CandidateIntake() {
           src={embedLink}
           frameBorder="0"
           width="100%"
-          height="8680"
+          height="7235"
```

Comment:
This is because the form length is dynamic and we keep changing it? Is there a more flexible way to do this?